origami.el

```diff
@@ -682,24 +683,23 @@ The fold node opened will be the deepest nested at POINT."
 
 ;;;###autoload
 (defun origami-open-node-recursively-till-depth (buffer point depth &optional include-siblings)
-  "Opens childs from the node at point up to the given depth, and collapses all other.
+  "Opens childs from the node at POINT up to the given DEPTH, and collapses all other.
 
-`depth' is relative to the node at point. It can be passed
+DEPTH is relative to the node at point. It can be passed
 interactively as a numeric prefix argument.
 
-`include-siblings' will also open the siblings of the node at
+INCLUDE-SIBLINGS will also open the siblings of the node at
```

Comment:
Cool, I was about to fix this myself. Thanks! :)

Reply:
Thanks for your fast response :)

I see these symbols are defined in a let form in the parsers. If I compile the parsers-file it without lexical-binding, I can reproduce your error. Did you by any change compile it without the `lexical-binding: t` at the top?

Reply:
Thanks for your fast response :)

I see these symbols are defined in a let form in the parsers. If I compile the parsers-file it without lexical-binding, I can reproduce your error. Did you by any change compile it without the lexical-binding: t at the top?

Reply:
Ah. I would prefer lexical-binding unless there is a good reason to not having it.

Could you point me where have you change that causes this issue? I couldn't find the code that may cause this error. 😕

Reply:
I agree, and didn't change anything about it in these commits, as far as I'm aware. It just had the suspicion that in your testing, you might have compiled without lexical-binding accidentally (like eval the code in a buffer where local variable lexical-binding was nil), and thus saw that error.

Are you sure that's not the case? I couldn't reproduce your error in another way so far.

Reply:
Ah, I see what I have missed! Thanks! :)